cpp/include/Ice/FacetMap.h

```diff
@@ -1,20 +1,19 @@
 //
 // Copyright (c) ZeroC, Inc. All rights reserved.
 //
-//
-
-#include <Ice/ObjectF.h>
-#include <map>
 
 #ifndef ICE_FACET_MAP_H
-#    define ICE_FACET_MAP_H
+#define ICE_FACET_MAP_H
+
+#include "ObjectF.h"
+#include <map>
 
 namespace Ice
 {
     /**
      * A mapping from facet name to servant.
      */
-    using FacetMap = ::std::map<::std::string, std::shared_ptr<Object>>;
+    using FacetMap = std::map<std::string, ObjectPtr>;
```

Comment:
should we use `std::lesss<>` in case we want to lookup with a string_view?

Reply:
I guess we could, but I'd rather leave this for another PR. Same question for the more frequently used Context:

```cpp
// generated code

/**
 * A request context. <code>Context</code> is used to transmit metadata about a request from the server to the client,
 * such as Quality-of-Service (QoS) parameters. Each operation on the client has a <code>Context</code> as its
 * implicit final parameter.
 */
using Context = ::std::map<::std::string, ::std::string>;
```